Exiled.Events/EventArgs/Scp939/LungingEventArgs.cs

```diff
@@ -7,37 +7,36 @@
 
 namespace Exiled.Events.EventArgs.Scp939
 {
+    using System;
+
     using API.Features;
     using Interfaces;
 
     /// <summary>
     ///     Contains all information before SCP-939 uses its lunge ability.
     /// </summary>
-    public class LungingEventArgs : IPlayerEvent, IDeniableEvent
+    public class LungingEventArgs : IPlayerEvent
     {
         /// <summary>
         ///     Initializes a new instance of the <see cref="LungingEventArgs" /> class.
         /// </summary>
         /// <param name="player">
         ///     <inheritdoc cref="Player" />
         /// </param>
-        /// <param name="isAllowed">
-        ///     <inheritdoc cref="IsAllowed" />
-        /// </param>
-        public LungingEventArgs(ReferenceHub player, bool isAllowed = true)
+        public LungingEventArgs(Player player)
```

Comment:
breaking change?

Reply:
no, the IsAllowed property is still here

Reply:
> no, the IsAllowed property is still here, but it will not work since we can't deny the event

ReferenceHub can be cast to Player?

Reply:
is the same thing

Reply:
I mis-read a bit. nvm.

Reply:
I misread